dist/rollbar.js

```diff
@@ -254,10 +254,7 @@
 	  // when the rate limit is reached.
 	  _topLevelNotifier = _topLevelNotifier || this;
 	
-	  var protocol = window.location.protocol;
-	  if (protocol.indexOf('http') !== 0) {
-	    protocol = 'https:';
-	  }
+	  var protocol = 'https';
```

Comment:
Looks like this variable is only used one place, the very next line. Any reason not to inline it in the next line?